frontstage_api/common/request_handler.py

```diff
@@ -10,12 +10,12 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def request_handler(method, url, auth=None, headers=None, json=None, data=None, fail=True):
+def request_handler(method, url, auth=None, headers=None, json=None, data=None, files=None, fail=True):
```

Comment:
I still think the value of having this wrapper doesn't warrant the overhead I think it's adding.

We are adding more and more arguments to support different use cases thus it's looking more like [requests](http://docs.python-requests.org/en/master/) api and less of an abstraction. Where we are passing in the `method` parameter we could just be defining the method name. We could also catch `RequestException` in the `error_handling` of flask and then we would be writing roughly the same amount of code.

Reply:
I agree, this has kind of been left over from an old way of organising things in the api that we aren't using anymore. It's used across the api though so probably best to deal with after this PR? We're making a list of the improvements we want to make across the repo once we have the functionality in